src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMSourceBuilder.java

```diff
@@ -74,7 +73,7 @@ public BitbucketSCMSourceBuilder(@CheckForNull String id, @NonNull String server
                                      @NonNull String repoName, @CheckForNull String mirrorId) {
         super(BitbucketSCMSource.class, repoName);
         this.id = id;
-        this.serverUrl = BitbucketEndpointConfiguration.normalizeServerUrl(serverUrl);
+        this.serverUrl = serverUrl;
```

Comment:
it was declared nonnull normalizeServerUrl is useless to default empty to bitbucket cloud